WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Customs/WooShippingCustomsItem.swift

```diff
@@ -0,0 +1,201 @@
+                    Spacer()
+                    Button {
+                        // TODO: Add information
+                    } label: {
+                        Image(systemName: "info.circle")
+                            .foregroundColor(Color(.wooCommercePurple(.shade60)))
+
+                    }
+                }
+                    .subheadlineStyle()
+
+                Button {
+                    isShowingCountries = true
+                } label: {
+                    HStack {
+                        Text(viewModel.originCountry.name)
+                            .bodyStyle()
+                        Spacer()
+                        Image(systemName: "chevron.up.chevron.down")
+                    }
+                    .padding()
+                }
+                .roundedBorder(cornerRadius: Layout.borderCornerRadius, lineColor: Color(.separator), lineWidth: Layout.borderLineWidth)
+            }
+            .padding(.leading, Layout.extraPadding)
+            .padding(.trailing, Layout.extraPadding)
+            .padding(.bottom, Layout.extraPadding)
+        })
+        .roundedBorder(cornerRadius: Layout.borderCornerRadius, lineColor: Color(.separator), lineWidth: Layout.borderLineWidth)
+        .safariSheet(isPresented: $isShowingHSTarrifInfoWebView, url: viewModel.hsTariffURL)
+        .sheet(isPresented: $isShowingCountries, content: {
+            NavigationStack {
+                SingleSelectionList(title: Localization.originCountryTitle,
+                                    items: viewModel.allCountries,
+                                    contentKeyPath: \.name,
+                                    selected: $viewModel.originCountry)
+            }
+            .wooNavigationBarStyle()
+        })
+    }
+}
+
+extension WooShippingCustomsItem {
+    enum Localization {
+        static let descriptionTitle = NSLocalizedString("wooShipping.customsItems.description",
+                                              value: "Description",
+                                              comment: "Title for the customs items description text field for customs items")
+        static let HSTarriffNumber = NSLocalizedString("wooShipping.customsItems.hsTarriffNumber",
+                                                       value: "HSTarriffNumber",
+                                                       comment: "Title for the HS Tarriff Number text field for customs items")
+        static let HSTarriffNumberPlaceholder = NSLocalizedString("wooShipping.customsItems.hsTarriffNumber.placeholder",
+                                                       value: "Optional",
+                                                       comment: "Placeholder for the HS Tarriff Number text field for customs items")
+        static let HSTarriffNumberMoreInfo = NSLocalizedString("wooShipping.customsItems.hsTarriffNumber.moreInfoText",
+                                                       value: "More info about HS tarriff",
```

Comment:
I think there's a typo for this string in the design:

```suggestion
                                                       value: "More info about HS tariff",
```

Reply:
Done in f1f3cc6f37116ba09cc56a38877f5c8384e55b4c